include/machine/io.h

```diff
@@ -26,9 +26,9 @@ unsigned char getDebugChar(void);
 /* printf will result in output */
 void putchar(char c);
 #define kernel_putchar(c) putchar(c)
-word_t kprintf(const char *format, ...) VISIBLE FORMAT(printf, 1, 2);
+void kprintf(const char *format, ...) VISIBLE FORMAT(printf, 1, 2);
```

Comment:
It probably makes sense to remove puts given that it only returns 0 and cannot fail, but kprintf returns the number of characters written and is consistent with all printf style interfaces that I'm aware of and so I think it should remain like that.